### PR TITLE
Add social links, add announcement

### PR DESCRIPTION
**Pull request by jyc1264**
NOTE: Announcement will be commented out before merge

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <!DOCTYPE html>
 <html>
+  <!-- <div class="announcement">
```
Comment (jennxhong) [inline _includes/header.html:3]: This should go inside of the body, right above the navbar.

File: assets/css/styles.css [hunk 2]
```diff
@@ -96,6 +96,20 @@ a:hover,
   margin: 30px 0;
 }
 
+.announcement {
+  display: inline-block;
```
Comment (jennxhong) [inline assets/css/styles.css:100]: I tested this and you don't need display, height, margin, padding, vertical-align. Can you remove and re-test after you move the announcement to the right place in the DOM?

File: index.html [hunk 3]
```diff
@@ -41,6 +41,13 @@ <h2>Belong</h2>
       <strong>Real people. Genuine connections.</strong>
     </p>
     <p>From working professionals to parents. From nursery to post-grad. From casual fellowshipping to theology discussions. Discover a place to belong at Christ Central.</p>
+      <ul style="padding-left: 15px;">
+        <li>Family LIfe Ministry</li>
```
Comment (jennxhong) [inline index.html:45]: Life has a capital I.

File: assets/css/styles.css (hunk 2 above)
Comment (jyc1264) [reply assets/css/styles.css:100]: yep ur right, removed

**Pull request merged**